src/Symfony/Component/Process/ExecutableFinder.php

```diff
@@ -48,7 +46,7 @@ public function addSuffix($suffix)
      *
      * @return string|null The executable path or default value
      */
-    public function find($name, $default = null, array $extraDirs = [])
+    public function find(string $name, ?string $default = null, array $extraDirs = [])
```

Comment:
```suggestion
    public function find(string $name, string $default = null, array $extraDirs = [])
```

CS: If the parameter has `null` as default value, we omit the `?`.